build.gradle

```diff
@@ -4,16 +4,16 @@ plugins {
 }
 
 ext {
-  openTelemetryVersion = "1.16.0"
-  openTelemetryAlphaVersion = "1.16.0-alpha"
-  openTelemetryInstrumentationVersion = "1.16.0"
+  // these lines are managed by .github/scripts/update-sdk-version.sh
+  otelVersion = "1.16.0"
+  otelAlphaVersion = "1.16.0-alpha"
```

Comment:
removed separate instrumentation version similar to the contrib repo, I think it probably makes sense to keep this repo on the same version of both

Reply:
We'll update it on the javaagent release then?

Reply:
right (of course I forgot about this PR and so the update failed on the release just now)